src/utils.ts

```diff
@@ -438,6 +438,33 @@ export function parseTsAliases(basePath: string, paths: ts.MapLike<string[]>) {
   return result
 }
 
+const rootAsteriskImportRE = /^(?!\.{1,2}\/)([^*]+)$/
+export function isAliasGlobal(alias: Alias) {
+  return alias.find.toString() === rootAsteriskImportRE.toString()
+}
+
+export function importResolves(path: string) {
+  const files = [
+    '.js',
+    '.jsx',
+    '.ts',
+    '.tsx',
+    '.json',
+    '.mjs',
+    '.cjs',
+    '.d.ts',
+    '.vue',
+    '.vue.d.ts'
+  ]
+
+  for (const ext of files) {
+    if (existsSync(path + ext)) {
+      return true
+    }
+  }
+  return false
+}
```

Comment:
Not really sure if this is the best way to do this. This current solution assumes a lot and isn't really configurable though it might be enough at this point. I thought about using the ts program but didn't want to go that deep and I'm not really sure it'd even be useful.

Reply:
@qmhc any thoughts?

Reply:
Well, I'm not sure it's possible to use `ts.resolveModuleName` to implement this feature.

Reply:
Do you think this needs any more work to merge?

Reply:
I think we should add `.mts` and `.cts` extensions, and because we have provided the svelte resolver, we should also consider `.svelte` files.

Reply:
Ok, that's done!